Bigtable/src/RetryUtil.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable;
+
+use Google\Rpc\Code;
+
+/**
+ * Utility class for retries.
+ */
+class RetryUtil
```

Comment:
I'm wondering if we need this for any situations outside of the resumable streams, it seems we should be able to rely on the functionality in the GAPICs for normal retries. If that is the case - would you have any objection to working the ideas here into the `ResumableStream` class?